transactions/transactions_service.go

```diff
@@ -206,6 +206,14 @@ func (svc *transactionsService) SendPaymentSync(ctx context.Context, payReq stri
 		return nil, err
 	}
 
+	if time.Now().After(time.Unix(int64(paymentRequest.CreatedAt+paymentRequest.Expiry), 0)) {
```

Comment:
what happens if there is no expiry in the invoice?

Reply:
You can omit it from the invoice - in the docs it then says it should default to 3600 seconds. But decodepay handles this for us, nice!